@paulcbetts/system-idle-time@1.0.4 Node Gyp Rebuild error

## My Setup
<!-- INSTRUCTION: Answer the questions below about your current setup -->
 - __My OS name and version__: Ubuntu 14.04
 - __My current app version__: 2.4.0

<!--
 INSTRUCTION: Answer the questions below by checking items with x, like:
 - [x] Checked Item
-->

## Description

<!-- INSTRUCTION: Add more information about your issue and how to simulate the problem if it is possible -->

On electron app, when i run npm install i am getting error in @paulcbetts/system-idle-time@1.0.4


npm ERR! Linux 3.19.0-25-generic
npm ERR! argv "/usr/bin/nodejs" "/usr/bin/npm" "install" "@paulcbetts/system-idle-time"
npm ERR! node v6.9.5
npm ERR! npm  v3.10.10
npm ERR! code ELIFECYCLE

npm ERR! @paulcbetts/system-idle-time@1.0.4 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @paulcbetts/system-idle-time@1.0.4 install script 'node-gyp rebuild'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the @paulcbetts/system-idle-time package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-gyp rebuild
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs @paulcbetts/system-idle-time
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls @paulcbetts/system-idle-time
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /opt/lampp/htdocs/Rocket.Chat.Electron/npm-debug.log

--- a/README.md
+++ b/README.md
@@ -72,6 +72,13 @@ build-essential
 libevas-dev
 libxss-dev
 ```
+### Fedora Install
+You will need to install:
+```sh
+libX11
+libXScrnSaver-devel
+gcc-c++
+```
 
 #### Windows 7
 On Windows 7 you may have to follow option 2 of the [node-gyp install guide](https://github.com/nodejs/node-gyp#installation) and install Visual Studio

--- a/README.md
+++ b/README.md
@@ -27,16 +27,11 @@ npm start
 
 ## Structure of the project
 
-The application is split between two main folders...
+The sources is located in the `src` folder. Everything in this folder will be built automatically when running the app with `npm start`.
 
-`src` - this folder is intended for files which need to be transpiled or compiled (files which can't be used directly by Electron).
-
-`app` - contains all static assets (put here images, css, html etc.) which don't need any pre-processing.
+Stylesheets are written in `less` and are located in `src/stylesheets`. They will be build into a single `main.css` in the `app` folder.
 
 The build process compiles all stuff from the `src` folder and puts it into the `app` folder, so after the build has finished, your `app` folder contains the full, runnable application.
-
-Treat `src` and `app` folders like two halves of one bigger thing.
-
 
 ## The build pipeline
 
@@ -51,16 +46,18 @@ Side note: If the module you want to use in your app is a native one (not pure J
 
 ## Working with modules
 
-Thanks to [rollup](https://github.com/rollup/rollup) you can (and should) use ES6 modules for all code in `src` folder. But because ES6 modules still aren't natively supported you can't use them in the `app` folder.
+Thanks to [rollup](https://github.com/rollup/rollup) you can (and should) use ES6 modules for most code in `src` folder.
 
 Use ES6 syntax in the `src` folder like this:
 ```js
 import myStuff from './my_lib/my_stuff';
 ```
 
-But use CommonJS syntax in `app` folder. So the code from above should look as follows:
+The exception is in `src/public`. ES6 will work inside this folder, but it is limited to what Electron/Chromium supports. The key thing to note is that you cannot use `import` and `export` statements. Imports and exports need to be done using CommonJS syntax:
+
 ```js
-var myStuff = require('./my_lib/my_stuff');
+const myStuff = require('./my_lib/my_stuff');
+const { myFunction } =  require('./App');
 ```
 
 ## Issues with Install

--- a/README.md
+++ b/README.md
@@ -63,6 +63,22 @@ But use CommonJS syntax in `app` folder. So the code from above should look as f
 var myStuff = require('./my_lib/my_stuff');
 ```
 
+## Issues with Install
+
+### node-gyp
+Follow the installation instruction on [node-gyp readme](https://github.com/nodejs/node-gyp#installation).
+
+#### Ubuntu Install
+You will need to install:
+```sh
+build-essential
+libevas-dev
+libxss-dev
+```
+
+#### Windows 7
+On Windows 7 you may have to follow option 2 of the [node-gyp install guide](https://github.com/nodejs/node-gyp#installation) and install Visual Studio
+
 # Testing
 
 ## Unit tests

